src/main/java/dota/buff/controller/HeroController.java

```diff
@@ -0,0 +1,30 @@
+package dota.buff.controller;
+
+import dota.buff.model.HeroDTO;
+import dota.buff.service.HeroService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("api/v1/hero")
+public class HeroController {
+    private final HeroService heroService;
+
+    @Autowired
+    public HeroController(HeroService heroService) {
+        this.heroService = heroService;
+    }
+
+    @GetMapping("/{id}")
```

Comment:
About Get or Post, I think when we don't have secure information we can use Get methods

Reply:
Okay